Chore: (Docs) Updates the env vars docs for sb env field (#17734)
storybookjs/storybook: docs/snippets/common/storybook-main-env-field-config.js.mdx

```js
// .storybook/main.js

module.exports = {
  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/addon-interactions',
  ],
  /* 
   * 👇 The `config` argument contains all the other existing environment variables.

>    * Either configured in an `.env` file or passed as command line arguments.

Reviewer: ```suggestion
   * Either configured in an `.env` file or configured on the command line.
```
